galaxy_importer/ansible_test/runners/__init__.py

```diff
@@ -31,7 +31,7 @@ def get_runner(cfg):
     if not cfg.run_ansible_test:
         return None
 
-    if not cfg.infra_pulp:
+    if not cfg.infra_local_image:
```

Comment:
since we now have a config value for this scenario, we can say this (or whatever the final config param name is)
```
if cfg.ansible_test_local_image:
    return LocalImageTestRunner
```